### DISCUSSION TITLE
Steering Sensitivity Curve

### DISCUSSION THREAD

**Discussion by inventor7777** [EdgeTX/edgetx#4645] [Q&A] [answered]
Hey everyone! I'm not quite sure how to use curves yet. I tried to do this a few days ago and it miserably failed 🥲

How do I make a curve that makes the steering control less sensitive on the lower end? 

For example. right now if I turn the steering wheel 5% and the steering servo turns 5%. How do I make it so when the transmitter wheel is turned 5%, the servo turns only 2%, 10% to 5%, and then leveling out near the middle of the spectrum?

Thank you for your help!

In addition, is it possible to regulate this curve based on throttle position? So that it's even more granular at top speed..?

**Comment by pfeerick** [comment]
Decreased sensitivity in the middle without compromising full movement could be done with expo, but perhaps it would be better with a curve. To make it so you have throttle basted limitation on the steering, probably need to look adding another mix line reduces the effect of the steering as you increase the throttle... Personally I think I just limited the steering weight (thus movement) with a pot because I was just lazy, but having it linked to the throttle position + with decreased sensitivity in the middle is the way to go... just can't remember offhand if I started working on that myself. 

There is a conversation on the discord server about staged throttle speed that might be of interest... https://discord.com/channels/839849772864503828/1197746507399569469

**Reply by inventor7777** [reply]
Thank you very much for the reply! 

I tried adding a mix to the steering channel to limit it based on throttle, but it just shows as a "+-" and simply turns the wheels when the throttle is pulled rather than limiting the steering. What am I doing wrong here?

I tried to include a screenshot from the radio (RadioMaster MT12) but it won't let me screenshot a settings page. :(

**Reply by pfeerick** [reply, marked as answer]
I'll double check, but you you should be able to screenshot any page?

To make it work, would probably need to change it to a "multiply" mix, rather than the default "Add", but then more fiddling is probably needed with weight and offset. Maybe even another mix line based on the throttle to have the input you want for the steering correction.

**Reply by inventor7777** [reply]
That works great! (Mostly) The only problem is that no matter what I try, the steering just freezes at zero when the throttle is maxed out. I've tried messing with offsets and such, but no luck. I'll keep trying but if you know why it does that let me know! 

And I discovered that i'd forgotten to set the screenshot function to global, which disabled it under the new model I'm working on....so it's working fine now.

This is my current setup (Each offset is the opposite of the number shown on the mix)
![screen-2024-02-16-153014](https://github.com/EdgeTX/edgetx/assets/142270525/b27cb64f-5900-4a91-96e9-c43e2a94c683)
